data/persistence/src/commonMain/sqldelightTimeline/social/androiddev/common/timeline/Timeline.sq

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE TimelineItem (
+  type TEXT NOT NULL,
+  remoteId Text NOT NULL PRIMARY KEY,
+  uri Text NOT NULL,
+  createdAt Text NOT NULL,
+  content Text NOT NULL,
+  accountId Text,
+  visibility Text NOT NULL,
+  sensitive INTEGER AS Boolean DEFAULT 0,
+  spoilerText Text NOT NULL,
+  avatarUrl Text NOT NULL,
+  accountAddress Text NOT NULL,
+  applicationName Text NOT NULL,
+  userName Text NOT NULL,
+  repliesCount INTEGER,
+  favouritesCount INTEGER,
+  reblogsCount INTEGER
+);
+
+insertFeedItem:
+INSERT OR REPLACE INTO TimelineItem
+VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?);
```

Comment:
I think it's safer to use `?;` instead of `(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?);` and insert via the TimelineItem model.